update.sh

```diff
@@ -15,6 +15,10 @@ git pull origin main
 git checkout main
 cd ..
 
+echo "update commit-msg"
+cp .github/hooks/commit-msg .git/hooks/commit-msg
```

Comment:
this should be `cp .github/hooks/commit-msg ../.git/hooks/commit-msg`

Reply:
is it because of symbolic link?